open_academy/wizard/open_academy_wizard.py

```diff
@@ -0,0 +1,22 @@
+from odoo import models, fields
+
+
+class Wizard(models.TransientModel):
+    _name = 'openacademy.wizard'
+    _description = 'Register attendees'
+
+    attendee_ids = fields.Many2many('res.partner', string="Attendees")
+
+    def _default_session(self):
+        return self.env['openacademy.session'].browse(self._context.get('active_id'))
```

Comment:
This method should be after fields see guidelines https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#37models
```suggestion
    def _default_session_id(self):
```

When you move it you need to change the field and use lambda function in the default parameter to find the method.
`default=lambda self: self._default_session_id()`